### PR TITLE
Run black linter and enforce going forward

### PR DESCRIPTION
**Pull request by asherf**
Basically a working copy of https://github.com/spulec/moto/pull/2290

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,9 +13,9 @@ init:
 	@pip install -r requirements-dev.txt
 
 lint:
-	flake8 moto
+	black --check moto/ tests/
 
-test: lint
```
Comment (asherf) [inline Makefile]: because we run it via travis and only in python 3.6 and up (black doesn't support older python versions.)

**Comment by asherf** [comment]
@spulec @mikegrima friendly ping... ;-)

**Comment by spulec** [comment]
Thanks for the work on this; it looks great!

My one concern is that this will result in a lot of people making PRs, having tests pass locally, and then Travis fails because of linting. Removing `lint` from `make test` also makes this worse. Any ideas here to give people feedback locally?

**Comment by asherf** [comment]
> Thanks for the work on this; it looks great!
> 
> My one concern is that this will result in a lot of people making PRs, having tests pass locally, and then Travis fails because of linting. Removing `lint` from `make test` also makes this worse. Any ideas here to give people feedback locally?

@spulec how about leaving the test target as is and adding a ‘test-only’ target that will get called from travis.yaml ?

**Comment by spulec** [comment]
That seems like a good plan to me.

Apologies for the merge conflicts that have resulted from me merging some other PRs.

**Comment by asherf** [comment]
@spulec updated.

**Comment by spulec** [comment]
Awesome. Thanks for all of the work!

**Pull request merged**